doc/source/whatsnew/v0.18.1.txt

```diff
@@ -179,6 +179,43 @@ New Behavior:
     # Output is a DataFrame
     df.groupby(pd.TimeGrouper(key='date', freq='M')).apply(lambda x: x[['value']].sum())
 
+Changes in ``read_csv`` errors
```

Comment:
this sub-section should be in API changes.

add a direction above for referncding this: ``.._whatsnew.......``